add_gem_package.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash -e
 
 usage() {
-	echo "Usage: $0 GEM_NAME [TEMPLATE [PACKAGE_SUBDIR [KOJI_TAG]]]"
+	echo "Usage: $0 GEM_NAME [TEMPLATE [PACKAGE_SUBDIR [REPO]]]"
 	echo "Valid templates: $(ls gem2rpm | sed 's/.spec.erb//' | tr '\n' ' ')"
-	./list_koji_tags
+	echo "Valid repos: foreman-el8 foreman-plugins-el8 katello-el8 foreman-client"
```

Comment:
I still think this should read
```suggestion
	echo "Valid repos: foreman-el8 foreman-plugins-el8 katello-el8 foreman-client-el8"
```

because in L64 you call `add_gem_to_comps $REPO` and `foreman-client` doesn't exist as a comps file and in L68 you match on `foreman-client-*` which requires the second dash.